src/components/404/404.jsx

```diff
@@ -1,27 +1,13 @@
-import React from "react"
-
-import Container from "./Container"
-import Wrapper from "./Wrapper"
-import Image from "./Image"
-import Text from "./Text"
-import Button from "./Button"
-
-import NotFoundImg from "../../assets/images/designs/not-found.svg"
-
-const home = () => {
-  window.location = "/"
+import React, { useEffect } from "react"
+import { navigate } from "gatsby"
+
+// (2021-11-24) - The navigate function will be removed in the next major release of Gatsby.
+const NotFound = () => {
+    useEffect(() => navigate("/"))
+    return (
```

Comment:
Nit: make one line

Reply:
We can't return this in one line I believe.

Reply:
The returned JSX e.g. `return <></>`